src/components/DataArchives/DataArchives.js

```diff
@@ -89,14 +89,15 @@ const styles = theme => ({
     marginTop: '2rem'
   }
 });
-const sensors = 'http://api.airquality.codeforafrica.org/v1/sensor/{sensorID}/';
-const query = 'http://api.airquality.codeforafrica.org/v1/filter/{query}';
-const data = 'http://api.airquality.codeforafrica.org/v1/data/';
-const now = 'http://api.airquality.codeforafrica.org/v1/now/';
+// const sensors = 'https://api.sensors.africa/v1/sensor/{sensorID}/';
+const query = 'https://api.sensors.africa/v1/filter?city=&country=&type=';
+const data = 'https://api.sensors.africa/v1/data/';
+const now = 'https://api.sensors.africa/v1/now/';
 const type = '{sensor type}';
-const area = '{lat, lon, distance}';
-const box = '{lat1, lon1, lat2, lon2}';
+const city = '{city}';
 const countryCode = '{country code}';
```

Comment:
We don't use country codes, although this should change and we  should start using country codes. But until then, set this to `{country name}`. 


All previous sensors before me have locations country as name instead of code, we should sync up next week to change them all to country code.

cc @kilemensi